wp-content/civi-extensions/goonjcustom/Civi/GoonjInitiatedEventsService.php

```
@@ -38,6 +42,199 @@ public static function getSubscribedEvents() {
+    if (in_array($paymentMethod, ['Cash', 'Check'], TRUE)) {
+      $update->addValue('Goonj_Events_Outcome.Cash_Contribution', max(0, $cashCurrent - $amount));
+    }
+
+    $update->execute();
+
+  } catch (\Exception $e) {
+    \Civi::log()->error("Failed to update monetary contribution delete stats for event: " . $e->getMessage());
+  }
+}
+
+public static function handleEventMaterialContributionDelete(string $op, string $objectName, $objectId, &$params) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$params'. (Unused Code Rules)

(UnusedFormalParameter)